ante/evm/09_increment_sequence.go

```diff
@@ -22,6 +23,12 @@ func IncrementNonce(
 	// we merged the nonce verification to nonce increment, so when tx includes multiple messages
 	// with same sender, they'll be accepted.
 	if txNonce != nonce {
+		if txNonce > nonce {
```

Comment:
nit: can we update the variable name for `nonce` in this function to be `currentNonce` or something similar for readability